scripts/cmake/tools_protoc.cmake

```diff
@@ -0,0 +1,70 @@
+defold_log("tools_protoc.cmake:")
+
+# Ensure that protoc is the one shipped with the Defold SDK (tmp/dynamo_home)
+# Layout used by Defold packages:
+#   ${DEFOLD_SDK_ROOT}/ext/bin/<host-platform>/protoc
+#   ${DEFOLD_SDK_ROOT}/bin/ddfc_cxx (protoc plugin)
```

Comment:
An example of our troubles in the past, users accidentally using their locally installed protoc.